docs/src/main/sphinx/glossary.rst

```diff
@@ -0,0 +1,195 @@
+
+JKS
+    Java KeyStore, the system of public key cryptography supported as one part
+    of the Java security APIs. The legacy JKS system recognizes keys and
+    certificates stored in *keystore* files, typically with the ``.jks``
+    extension, and relies on a system-level list of CAs in *truststore* files
+    installed as part of the current Java installation.
+
+Terms K-O
+---------
+
+Key
+    A cryptographic key specified as a pair of public and private keys.
+
+.. _glossLB:
+
+Load Balancer (LB)
+    Software or a hardware device that sits on a network's outer edge or
+    firewall and accepts network connections on behalf of servers behind that
+    wall. Load balancers carefully manage network traffic, and can accept TLS
+    connections from incoming clients and pass those connections transparently
+    to servers behind the wall.
+
+.. _glossObjectStorage:
+
+Object Storage
+    `Object storage <https://en.wikipedia.org/wiki/Object_storage>`_ is a file
+    storage mechanism, that stores data in a random access manner as opposed to
+    hierarchical filesystems or sector-based block storage. In the context of
+    Trino, this commonly refers to
+    `cloud storage <https://en.wikipedia.org/wiki/Object_storage#Cloud_storage>`_
+    technologies such as `Amazon S3 <https://aws.amazon.com/s3>`_,
+    `Google Cloud Storage <https://cloud.google.com/storage>`_,
+    `Azure Blob Storage <https://azure.microsoft.com/en-us/products/storage/blobs>`_,
+    and others. Despite the nomenclature of cloud storage, there are also local
+    storage options that follow the same S3 format like `MinIO <https://min.io/>`_
+    and `Ceph <https://docs.ceph.com>`_. Object storage became a popular
+    replacement to :ref:`HDFS <glossHDFS>`.
+
+.. _glossOpenSource:
+
+Open-source
+    Open source can relate to many things, but generally refers to
+    `open-source software <https://en.wikipedia.org/wiki/Open-source_software>`_.
+    This is software that has the source code made available for others to see
+    and use. The usage will vary depending on the license that the software is
+    licensed under. Trino is licensed under the
+    `Apache license <https://en.wikipedia.org/wiki/Apache_License>`_, and
+    therefore maintained by a community of contributors from all across the
+    globe.
+
+Terms P-T
+---------
+
+PEM
+    Privacy-Enhanced Mail, a syntax for private key information, and a content
+    type used to store and send cryptographic keys and certificates. PEM format
+    can contain both a key and its certificate, plus the chain of certificates
+    from authorities back to the root :ref:`CA <glossCA>`, or back to a CA
+    vendor's intermediate CA.
+
+.. _glossPKCS12:
+
+PKCS #12
+    A binary archive used to store keys and certificates or certificate chains
+    that validate a key. `PKCS #12 <https://en.wikipedia.org/wiki/PKCS_12>`_
+    files have ``.p12`` or ``.pfx`` extensions.
+
+Presto and PrestoSQL
+    Old name for Trino.
```

Comment:
Seems almost comically concise. Can we at least link to one of the many pages/resources that announced the change and why, with a "To learn more about the name change to Trino, read..." type sentence?